src/index.js

```diff
@@ -4,8 +4,8 @@ import wrappers from './wrappers';
 const debug = makeDebug('feathers-rest');
 
 function formatter(req, res, next) {
-  if(!res.data) {
-    next();
+  if(typeof res.data === 'undefined') {
```

Comment:
Why do you check for typeof being `undefined` instead of just checking for `undefined`?